doc/Notebook_review_guide.md

```diff
@@ -0,0 +1,120 @@
+
+9. After merging all reviews, merge the notebook review branch into the primary branch (main or dev).
+
+## Instructions Template
+
+Sections will be titled in *italics* and inline suggestions will be bolded.
+
+*Topics*
+
+**What follows are the instructions for review and feedback on the notebook "notebook_name.ipynb". The objective of this notebook is as a tutorial for Python users to work with the following topics:**
+
+Review the notebook and pick out 4-7 topics that are important for reviewers to understand when reviewing the notebook.
+
+*Prerequisites to perform review*
+
+**The following are needed for performing this review:**
+
+What platform will the review be performed on and does the reviewere have access? Who does the reviewer contact to get access? Are there any other accounts the reviewer will need to have?
+
+*Instructions to perform review*
+
+This section will lay out the steps to completing the review. Some important steps are:
+
+1. How to create a branch (if a member of repository organization) or fork the project (if not a member).
+2. How to setup the environment on the platform the review will be done on. If extra instructions are needed to setup a custom environment, be sure to include an environment YML to do so.
+3. How to format feedback in notebook. A common way is "**[first name] comment/edited text/edited code"**.
+
+*Review types and descriptions*
+Title the review types in bold and add some questions/comments to help the reviewer understand the scope of the type.
+
+*Where/how to submit feedback*
+
+Include instructions on opening a PR with specifics for those who are part of the organization and those who aren't. Include the branch to compare to in the PR, who to assign as a reviewer, and the date to submit feedback by.
+
+## Instructions Example (GitHub)
+
+What follows are the instructions for review and feedback on the notebook "CONUS404_DRB_Data_Prep.ipynb". The objective of this notebook is as a tutorial for newer Python users to work with the following topics:
+
+CONUS404 dataset
+exploratory spatial data analysis (ESDA)
+data preparation of multiple datasets for comparative analysis, including:
+temporal extent and step
+geographic extent
+parallel computing
+accessing data from a variety of sources, including:
+cloud storage
+API calls
+FTP calls
+The following are needed for running this notebook:
+
+Access to the [ESIP-Nebari on AWS](https://nebari.esipfed.org/). If you do not have access to this, please email Andrew at alaws@contractor.usgs.gov and he will get you credentials.
+Github account (though this can be worked around if need be, just let Andrew know)
+Set Up
+
+Navigate to the [hytest-org repo](https://github.com/hytest-org/hytest)
+If you are a member of the `hytest-org` GH organization, create a branch named "initials-NB-review"
```

Comment:
Didn't one of the last reviewers try to create a branch with that exact name? "initials-NB-review"...   ??

"create a branch into which your suggestions will go -- such as xx-NN-review, where 'xx' is your initials.